lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -1124,6 +1124,16 @@ export class StandardController implements IController {
         }
     }
 
+    /**
+     * Creates a cache interaction client to clear broswer cache.
+     * @param logoutRequest
+     */
+    async clearCache(logoutRequest?: EndSessionRequest): Promise<void> {
```

Comment:
I wonder if we should accept a smaller request type here as many of the params aren't relevant. i.e. ClearCacheRequest with optional members correlationId and account

Reply:
Okay, that makes sense. I can create another request type.